style.scss

```diff
@@ -121,4 +131,57 @@ $medium: 768px;
     width: 100vw;
     height: 100vh;
     background-color: blue;
+}
+
+footer {
+    height: 353px;
+    background: #87AABC 0% 0% no-repeat padding-box;
+    opacity: 1;
+    padding: 47px 53px 81px 53px;
+    display: flex;
+    flex-direction: column-reverse;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+}
+
+.footerNav {
+    list-style: none;
+    margin-bottom: 35px;
+    & li {
```

Comment:
Is it necessary to use the ampersand inside of the .footerNav class? I think it's okey to just write li.

Reply:
You're right, my mistake!

Reply:
You are right! I didn't know you could omit the ampersand. I'll merge for now but keep that in mind in the future.